spark/dl/src/main/scala/com/intel/analytics/bigdl/example/mkldnnint8/GenerateInt8Scales.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.example.mkldnnint8
+
+import com.intel.analytics.bigdl.dataset.image.CropCenter
+import com.intel.analytics.bigdl.dataset.{DataSet, Sample, SampleToMiniBatch}
+import com.intel.analytics.bigdl.nn.Module
+import com.intel.analytics.bigdl.transform.vision.image.augmentation.{ChannelScaledNormalizer, RandomCropper, RandomResize}
+import com.intel.analytics.bigdl.transform.vision.image.{ImageFeature, ImageFrameToSample, MatToTensor, PixelBytesToMat}
+import com.intel.analytics.bigdl.utils.Engine
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.SparkContext
+
+object GenerateInt8Scales {
+  Logger.getLogger("org").setLevel(Level.ERROR)
+  Logger.getLogger("akka").setLevel(Level.ERROR)
+  Logger.getLogger("breeze").setLevel(Level.ERROR)
+
+
+  import Utils._
+
+  def main(args: Array[String]): Unit = {
+    testParser.parse(args, new TestParams()).foreach { param =>
+      val conf = Engine.createSparkConf().setAppName("Quantize the model")
+        .set("spark.akka.frameSize", 64.toString)
+        .set("spark.task.maxFailures", "1")
+      val sc = new SparkContext(conf)
+      Engine.init
+
+      // We set partition number to be node*core, actually you can also assign other partitionNum
+      val partitionNum = Engine.nodeNumber()
+      val imageFrame = DataSet.SeqFileFolder.filesToImageFrame(param.folder, sc, 1000,
+        partitionNum = Option(partitionNum))
+      val transformer =
```

Comment:
add comments that the transformer should be the same as that in validation during training